src/app/notifications-page/notifications-list/notifications-list.component.ts

```diff
@@ -216,15 +216,29 @@ export class NotificationsListComponent implements OnInit {
         return null;
       }
 
-      const isDiamond = cctMeta.DiamondLevel > 0;
-      result.icon = isDiamond ? "icon-diamond fc-blue" : "fas fa-paper-plane fc-blue";
-      const diamondText = isDiamond
-        ? "<b>" + cctMeta.DiamondLevel.toString() + " diamond" + (cctMeta.DiamondLevel > 1 ? "s" : "") + "</b>, worth "
-        : "";
-      result.action = `${actorName} sent you ${diamondText} <b>${this.globalVars.nanosToBitClout(
-        cctMeta.CreatorCoinToTransferNanos,
-        6
-      )} ${cctMeta.CreatorUsername} coins`;
+      if (cctMeta.DiamondLevel) {
+        result.icon = "icon-diamond fc-blue";
+        let postText = "";
+        if (cctMeta.PostHashHex) {
+          const post = this.postMap[cctMeta.PostHashHex];
+          if (!post) {
+            return null;
+          }
+          const truncatedPost = _.truncate(_.escape(`${post.Body} ${post.ImageURLs?.[0] || ""}`));
```

Comment:
can we move this to a method to avoid code duplication?

looks great otherwise!